source/languages/en/riak/dev/taste-of-riak/haskell.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "Taste of Riak: Haskell"
+project: riak
+version: 1.4.0+
+document: guide
+toc: true
+audience: intermediate
+keywords: [developers, client, haskell]
+---
+
+If you haven't set up a Riak Node and started it, please visit the
+[[Prerequisites|Taste of Riak: Prerequisites]] first.
+
+To try this flavor of Riak, a working installation of [Haskell](http://www.haskell.org/platform/) (ideally the Haskell Platform) with [Cabal](http://www.haskell.org/haskellwiki/Cabal/How_to_install_a_Cabal_package) is required.
+
+### Client Setup
+
+The [riak-haskell-client](https://github.com/markhibberd/riak-haskell-client) is a
+community-maintained Riak client for Haskell.
+
+We're assuming you have a working install of ghc and cabal, if not consult your
+OS documentation on how to make that happen.
+
+First, clone the `taste-of-haskell` sample project from Github:
+
+```
+git clone git@github.com/tmcgilchrist/taste-of-haskell.git
+cd taste-of-ocaml
+make
+```
+
+Running make will setup a cabal sandbox and install all the necessary
+dependencies to get `riak-haskell-client` running.
+
+The `src` directory contains a single file titled `Main.hs`.
+
+The sample code tries connecting to 127.0.0.1 port 8087 by default. If you setup
+a local Riak cluster using the [[five minute install]] method, change
```

Comment:
Looks like it can be removed. I'll delete etc and fix it up.
I based this off the ocaml taste of where they change the ports around, but the code below looks like it uses the right port.